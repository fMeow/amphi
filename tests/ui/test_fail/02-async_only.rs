use amphi::amphi;

#[amphi(async_only)]
mod amphi{
    pub async fn my_fn() -> bool {
        true
    }
}

fn main() {
    use self::sync::my_fn;
}
