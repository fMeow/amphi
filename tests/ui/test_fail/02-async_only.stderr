error[E0432]: unresolved import `self::sync`
  --> $DIR/02-async_only.rs:11:15
   |
11 |     use self::sync::my_fn;
   |               ^^^^
   |               |
   |               unresolved import
   |               help: a similar path exists: `async_std::sync`
